README.md

```diff
@@ -25,6 +25,11 @@ through that README and the corresponding tests in parallel.
 
 Install GraphQL.js from npm
 
+With yarn:
+```sh
+yarn add graphql
+```
+or alternatively using npm:
```

Comment:
Yes! Can we get blank lines before and after each fenced code block before you land this?